src/browser/workspace-api.ts

```diff
@@ -163,4 +165,16 @@ export class RestDevWorkspaceApi implements IDevWorkspaceApi {
     this.axios.post(`/apis/${projectApiGroup}/v1/${projectRequestId}`, projectRequestModel(namespace));
   }
 
+  private async waitForProjectToBeReady(namespace: string): Promise<void> {
+    let attempt = 0;
+    let request = await this.doesProjectExist(namespace);
+    while (attempt < this.projectRequestMaxAttempts && !request) {
+      request = await this.doesProjectExist(namespace);
+      attempt += 1;
+    }
+    if (attempt === this.projectRequestMaxAttempts) {
+      throw new Error('Project could not be found');
```

Comment:
Project $namespace is not initialized in 10 seconds.

^ I'm not sure if it's 10 seconds or just 10 tries which may be done in 1 second.